src/google-maps/map-info-window/map-info-window.scss

```diff
@@ -0,0 +1,5 @@
+.map-info-window-container {
+  .map-info-window-content {
+    display: none;
+  }
```

Comment:
All three of these comments are related. The purpose of display: none is to make sure that content users write inside the info window does not show up in the regular dom. It has this level of specificity because the container is not put into the info window with the content, so this style won't affect the display of the content inside the info window.